app/resources/admin.html.template

```diff
@@ -975,6 +955,32 @@
     </div>
   </fieldset>
 
+
+  <fieldset>
+    <legend>Notification settings</legend>
+      <label for="sms_number_to_repo">
```

Comment:
This is not a label for sms_number_to_repo. You can probably replace this ```<label>``` with ```<div>```.